### DISCUSSION TITLE
显示pdf名称改为显示条目名称

### DISCUSSION THREAD

**Discussion by Chikit-L** [volatile-static/Chartero#16] [Ideas]
能不能将items和时间线中的pdf名称改为条目名称呢，许多pdf文件zotero都是自动命名为full text pdf。在chartero中全部都是full text pdf，无法分别

**Comment by volatile-static** [comment]
请大家在这里讨论应该选用哪种命名

**Reply by Eliott-He** [reply]
我选条目名

**Reply by Comte0825** [reply]
我选条目，以为PDF名字的话，有的人可能没有设置重命名插件，导致很多”FULL PDF TEXT“

**Reply by YYde123** [reply]
有个问题，chartero装上后，在条目词云哪里，标题和作者无法获取数据啊

**Comment by CailongNie** [comment]
条目名

**Comment by wongtx** [comment]
条目名

**Comment by volatile-static** [comment]
感谢大家反馈，代码已修改，但这几天暂时不会发布新版，急用的同学可以自行打包安装

**Reply by wang-jh18-SVM** [reply]
请问现在发布的版本是否更新了这一功能呢？我的插件中依然是显示附件名，而非条目名
![image](https://github.com/volatile-static/Chartero/assets/57933327/a16995ec-f900-4cfb-9749-a7ab4a054e4d)

我的zotero版本是7.0.0-beta.54+6b996d4f9 (64-bit)；chartero版本是2.3.1

**Reply by volatile-static** [reply]
之前该的是旧版的图，现在的版本为了区分多个附件的条目，暂时还是采用附件名，之后有空会优化为树状显示

**Reply by wang-jh18-SVM** [reply]
好的，感谢回复
